rskj-core/src/main/java/org/ethereum/config/Constants.java

```diff
@@ -50,10 +50,12 @@
     private final boolean seedCowAccounts;
     private final int durationLimit;
     private final BlockDifficulty minimumDifficulty;
+    private final BlockDifficulty minimumDifficultyForRskip290 = new BlockDifficulty(new BigInteger("550000000"));
```

Comment:
This works, but it deviates from the way the other variables are set. I would consider setting it in the `testnet` constructor.

Reply:
Done. Updated.

Reply:
I might be wrong, but it seems to me that @joaquinlpereyra meant [this](https://github.com/rsksmart/rskj/pull/1654/files#diff-f405fc1a0176d94e7c3c2572e30caaadbd8b571f2cac6892358bb0948de6c96aR252) place